surfsense_backend/app/connectors/discord_connector.py

```
@@ -0,0 +1,264 @@
+"""
+Discord Connector
+
+A module for interacting with Discord's HTTP API to retrieve guilds, channels, and message history.
+
+Requires a Discord bot token.
+"""
+
+import logging
+import discord
```

Pylint (3.3.7)

[error] Unable to import 'discord'

(E0401)